src/game/objects/general/bridge.h

```diff
@@ -4,6 +4,8 @@
 
 #include <stdint.h>
 
+int32_t Bridge_GetOffset(ITEM_INFO *item, int32_t x, int32_t y, int32_t z);
+bool Bridge_OnDrawBridge(ITEM_INFO *item, int32_t x, int32_t z);
```

Comment:
I don't think we ever use `Bridge_OnDrawBridge` publicly, so this can be made `static` back.